--- /home/galloa/GitHub/work/elvis/srv/company_history.txt	2025-12-29 17:42:27.734655400 +0800
+++ /home/galloa/GitHub/work/elvis/srv/company_history.txt.tmp	2025-12-29 17:42:27.686711200 +0800
@@ -0,0 +1,14 @@
+Compass Group
+Techforce Personnel Pty.
+Sodexo
+Niche Resources Group
+Evolve Talent
+Cater Care Services
+Engineering Trades Australia
+Fetch Recruitment
+GEM Resourcing
+Pindari WA PTY LTD
+Connect Resources Pty Ltd
+Glencore
+Clough Projects
+Itch Recruitment
