--- /home/galloa/GitHub/work/elvis/srv/company_history.txt	2025-12-29 17:10:07.148263000 +0800
+++ /home/galloa/GitHub/work/elvis/srv/company_history.txt.tmp	2025-12-29 17:10:07.075872000 +0800
@@ -0,0 +1,14 @@
+Compass Group
+Techforce Personnel Pty.
+Cater Care Services
+Sodexo
+Evolve Talent
+Niche Resources Group
+Fetch Recruitment
+Engineering Trades Australia
+Pindari WA PTY LTD
+Itch Recruitment
+GEM Resourcing
+EASTERN GURUMA
+Hagstrom Drilling Pty Ltd
+Flanco Pty Ltd
